### DISCUSSION TITLE
Any alternatives to autotag?

### DISCUSSION THREAD

**Discussion by JamesSayegh** [NvChad/NvChad#1998] [Q&A] [answered]
i tried setting up autotag based off this issue  #https://github.com/windwp/nvim-ts-autotag/issues/64#issuecomment-1509722026 while autotag runs and loads fine it is not functional, any alternatives?

**Comment by siduck** [comment, marked as answer]
@Persona036 i personally use autotag, works fine. I dont think there are alternatives which are maintained currently,  my settings : 

```lua
  {
    "windwp/nvim-ts-autotag",
    event = "InsertEnter",
    config = function()
      require("nvim-ts-autotag").setup()
    end,
  }
  ```
